api_tests/institutions/views/test_institution_users_list.py

```diff
@@ -1,29 +1,42 @@
-from nose.tools import *  # flake8: noqa
-
-from tests.base import ApiTestCase
-from osf_tests.factories import InstitutionFactory, UserFactory
+import pytest
 
 from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    InstitutionFactory,
+    UserFactory,
+)
+
+@pytest.mark.django_db
+class TestInstitutionUsersList:
```

Comment:
Since, for now, it's just one test, the performance difference is barely noticeable. iirc Steve was in favor for not using fixtures if it's a single test and more tests are not likely to be added (say no todo in the file) @Nesiehr could you paste your flowdock exchange with Steve about this?